app/assets/stylesheets/views/_brexit_checker_account_signup.scss

```diff
@@ -0,0 +1,7 @@
+.govuk-experiment > .govuk-inset-text {
+  border-color: #1d70b8;
+}
+
+#account-signup {
```

Comment:
the form component didn't seem to accept margin-bottom as a parameter in the same way the links did.